core/src/main/java/org/frankframework/align/content/JsonDocumentContainer.java

```diff
@@ -34,14 +34,14 @@
 public class JsonDocumentContainer extends TreeContentContainer<JsonElementContainer>{
 	protected Logger log = LogUtil.getLogger(this.getClass());
 
-	private final String name;
+	private @Getter final String name;
 	private final boolean skipArrayElementContainers;
-	private boolean skipRootElement;
-	private final String attributePrefix = "@";
-	private final String mixedContentLabel = "#text";
+	private final boolean skipRootElement;
```

Comment:
If you cannot set this to anything but `false`, then it can just as well be removed.

Is it possible to set this from the Configuration.xml, or was this possibly set via reflection, in a way that didn't show up in IntelliJ?

Reply:
I don't understand your feedback. The `skipRootElement` can just be set using the constructor. It can be set to true or false...

Reply:
My bad, I don't know why that did not occur to me when I wrote that 🤦

Reply:
No problem.